At least one area of unit/integration tests will need to be examined and fixed. Tests we write typically aren't DRY and they raise AssertionErrors when expected values change. Here's the first part of Travis' traceback:
```
FAIL: test_cmd_work (arxcode.world.dominion.tests.TestGeneralDominionCommands)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/home/travis/build/Arx-Game/arxcode/world/dominion/tests.py", line 162, in test_cmd_work
    'You use 5 action points and have 50 remaining this week.|'
  File "/home/travis/build/Arx-Game/arxcode/server/utils/test_utils.py", line 130, in call_cmd
    return self.call(self.instance, args, msg, caller=self.caller, **kwargs)
```

Out of this, we can glean the method & Class containing the failure,  **test_cmd_work** in **TestGeneralDominionCommands**, and the line number of the module/file where it failed:
`File "/home/travis/build/Arx-Game/arxcode/world/dominion/tests.py", line 162, in test_cmd_work`

Then the AssertionError shows what the test was written to expect, and what it got instead:
```
AssertionError:
==============================Wanted message==================================
'You use 5 action points and have 50 remaining this week.|Your social clout combined with that of your protege reduces difficulty by 22.\nChar rolling intellect and economics. \nYou and Orgtest both gain 12,000 prestige.\nYou have increased the economic influence of Orgtest by 25.\nCurrent modifier is 0, progress to next is 1/100.'
=============================Returned message================================
'You use 5 action points and have 50 remaining this week.|Your social clout combined with that of your protege reduces difficulty by 22.\nChar rolling intellect and economics. \nYou and Orgtest both gain 8,000 prestige.\nYou have increased the economic influence of Orgtest by 25.\nCurrent modifier is 0, progress to next is 1/100.'
```
All we have to do here is change the test numbers to reflect the expected outcome your new formulae. Other tests/lines will probably fail in the same manner, especially in this method and maybe in world.fashion.tests. While Travis will catch them, that's time-consuming compared to running tests locally and fixing them all at once.
`evennia test --settings=test_settings --nomigrations`

--- a/world/dominion/tests.py
+++ b/world/dominion/tests.py
@@ -162,7 +162,7 @@ class TestGeneralDominionCommands(ArxCommandTest):
                       'You use 5 action points and have 50 remaining this week.|'
                       'Your social clout combined with that of your protege reduces difficulty by 22.\n'
                       'Char rolling intellect and economics. \n'
-                      'You and Orgtest both gain 12,000 prestige.\n'
+                      'You and Orgtest both gain 8,000 prestige.\n'
                       'You have increased the economic influence of Orgtest by 25.\n'
                       'Current modifier is 0, progress to next is 1/100.')
         self.call_cmd("/score orgtest2", "No match for an org by the name: orgtest2.")

